docs/attribution/0001-analytics.rst

```diff
@@ -8,273 +8,226 @@
-+=======================+===========================================================================+
-| ``data-cta-type`` *   | Link type (e.g. ``navigation``, ``footer``, or ``button``)                |
-+-----------------------+---------------------------------------------------------------------------+
-| ``data-cta-text``     | name or text of the link                                                  |
-+-----------------------+---------------------------------------------------------------------------+
-| ``data-cta-position`` | Location of CTA on the page (e.g. ``primary``, ``secondary``, ``header``) |
-+-----------------------+---------------------------------------------------------------------------+
+Link click is intended to track links that are of interest but not the focus of the page. Examples include links in paragraphs,
+lists, FAQs, supplemental content, or in a navigation menu. Links can go to or away from the site.
 
-For all links to accounts.firefox.com use these data attributes (* indicates a required attribute):
+The attribute ``data-link-text`` must be present to trigger the event.
 
-+-----------------------+----------------------------------------------------------------------------------+
-| Data Attribute        | Expected Value                                                                   |
-+=======================+==================================================================================+
-| ``data-cta-type`` *   | fxa-servicename (e.g. ``fxa-sync``, ``fxa-monitor``)                             |
-+-----------------------+----------------------------------------------------------------------------------+
-| ``data-cta-text``     | Name or text of the link (e.g. ``Sign Up``, ``Join Now``, ``Start Here``).       |
-|                       |                                                                                  |
-|                       | We use this when the link text is not useful, as is the case with many           |
-|                       | account forms that say, ``Continue``. We replace ``Continue`` with ``Register``. |
-+-----------------------+----------------------------------------------------------------------------------+
-| ``data-cta-position`` | Location of CTA on the page (e.g. ``primary``, ``secondary``, ``header``)        |
-+-----------------------+----------------------------------------------------------------------------------+
++-----------------------+------------------------------------------------------------------------------------------------+
+| Data Attribute        | Expected Value                                                                                 |
++=======================+================================================================================================+
+| ``data-link-text`` *  | Text or name of the link (e.g. ``Monitor``, ``Features``, ``Instagram (mozilla)``,             |
```

Comment:
**Question:** What is this asterisks referring to?

Reply:
It's a required attribute. I added a note in the docs.